frontend/src/components/MakerForm/MakerForm.tsx

```diff
@@ -371,6 +386,13 @@ const MakerForm = ({
     });
   };
 
+  const handleClickAmountRangeEnabled = function (
+    _e: React.ChangeEvent<HTMLInputElement>,
+    checked: boolean,
+  ) {
+    setAmountRangeEnabled(!checked);
```

Comment:
It might be the `!` is not needed? The `checked` bool comes as `false` when unchecked. This function seems to make the switch remain in their previous state.

Reply:
Resolved in the last commit 375eb495016decfc2bffd61172e8ab50fce5dc8a.